engine/src/main/java/org/terasology/rendering/nui/layers/ingame/ExtraMenuScreen.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2019 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.nui.layers.ingame;
+
+import org.terasology.crashreporter.CrashReporter;
+import org.terasology.engine.GameEngine;
```

Comment:
You've got unused imports in the file like this one - the IDE should mark this very clearly and Checkstyle will report those as style warnings. Please fix and be sure to do a Checkstyle pass to find such issues :-)